File: exceptionlibrary/src/main/java/com/armboldmind/exceptionlibrary/IExceptionSendDeveloperControllerAPI.java
package com.armboldmind.exceptionlibrary;

import retrofit2.Call;
import retrofit2.http.Body;
import retrofit2.http.POST;

public interface IExceptionSendDeveloperControllerAPI {
    @POST("/services/TD7B96B8Q/BK0GH7ENQ/H2eaqWo8QwtL5dGyZOd8Yctj")
    Call<Object> loadChanges(@Body ErrorModel model);
}

File: exceptionlibrary/src/main/java/com/armboldmind/exceptionlibrary/IExceptionSendDeveloperControllerAPI.kt
package com.armboldmind.exceptionlibrary

import retrofit2.Call
import retrofit2.http.Body
import retrofit2.http.POST
import retrofit2.http.Path

internal interface IExceptionSendDeveloperControllerAPI {
    @POST("/services/{apiA}/{apiB}/{apiC}")
    fun loadChanges(
        @Path("apiA") apiA: String,
        @Path("apiB") apiB: String,
        @Path("apiC") apiC: String,
        @Body model: ErrorModel?
    ): Call<Any?>?
}